### ISSUE TITLE
Invalid ePub generation

### ISSUE THREAD

**Issue by xogeny** [xogeny/ModelicaBook#44]
Investigate ePub formatting issues highlighted by http://validator.idpf.org/

**Comment by dietmarw** [comment]
The SVG issue might actually get fixed once we get #274 fixed.

**Comment by xogeny** [comment]
Is this resolved?

**Comment by dietmarw** [comment]
I ran the validator once more yesterday and still got load of warnings but since WE don't generating the pub but Sphinx does I don't see how we could solve this. So I suggest to close this one and rather have issues reported when people have real problems with the book versions.

**Comment by xogeny** [comment]
Is it possible to paste the warnings in here just so we have a record of them?  It may be that there is some kind of post-processing we can do to get rid of the warnings.  We are already inlining the math, so it could be something we do at the same time as the math (before wrapping it up in to the `.epub` file).

**Comment by dietmarw** [comment]
I still don't see much point in this but here are the log files for the two versions of epub: 
[epubimg.txt](https://github.com/xogeny/ModelicaBook/files/1479240/epubimg.txt)
[epubsvg.txt](https://github.com/xogeny/ModelicaBook/files/1479241/epubsvg.txt)

**Comment by xogeny** [comment]
Well, most of the image ones are because there is an attribute called `align` being placed on `<div>`s that shouldn't be there.  I tried using `tidy` (old version that comes with OSX) and `tidy-html5` (newer version, hosted on GitHub).  It turns out that `tidy-html5` ignored it until I turned on the `strict-tags-attributes` which caused it to recognize this was a problem, but not correct it. 😢 

I was hoping that I could find something to clean up the HTML after the fact.  But I've spent too much time on this already.  The idea of cleaning up is a good one, if we could find the right tool.
